x-pack/plugins/apm/new_e2e/cypress/integration/home.spec.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+describe('Home page', () => {
+  it('Redirects to service page with rangeFrom and rangeTo added to the URL', () => {
+    const now = new Date(Cypress.env('METADATA_END_DATE'));
+    cy.clock(now);
```

Comment:
Is `now` really the current timestamp? 

```suggestion
    const endDate = new Date(Cypress.env('METADATA_END_DATE'));
    cy.clock(endDate);
```